stylus/include/Home.styl

```diff
@@ -17,16 +17,17 @@
 		padding-bottom: 3rem
 	>div:nth-child(5), div:nth-child(6)
 		display: flex
-		justify-content: center
+		justify-content: space-evenly
```

Comment:
I wasn't aware of this until now, and it doesn't appear to be fully supported. Just go with `center`.

Reply:
@ScottAP108, give us a link regarding `space-evenly` support.

Reply:
Check out "Browser compatibility" at the bottom: https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content. I also read some people say it doesn't work right completely even when it is supported.